src/ui/DynamicFacet/DynamicFacetRange.ts

```diff
@@ -49,16 +49,27 @@ export class DynamicFacetRange extends DynamicFacet implements IComponentBinding
       localizedString: () => l('To'),
       section: 'CommonOptions'
     }),
+    field: ComponentOptions.buildFieldOption({ required: true, section: 'CommonOptions' }),
     /**
      * The string format to apply to the minimum and maximum value of each range displayed in the facet.
      *
-     * **Default:** [`number`]{@link DynamicFacetRangeValueFormat.number}
+     * **Default:** If the [`field`]{@link DynamicFacet.options.field} contains "date", the format will be [`date`]{@link DynamicFacetRangeValueFormat.date}.
```

Comment:
Rather than looking for the substring "date" in the field name (and thus wrongly interpreting words such as "candidate", "consolidate", and "mandate" as being date-related), perhaps we should check against a specific list of field names that we can safely assume are date fields (e.g., standard date fields in the index).